final-project/src/tier1/action_primitives.py

```diff
@@ -44,6 +44,8 @@ def straight_line_rotations(self):
                 print("matrix" + str(i+1))
                 print(matrix[i])
         # Convert to bezier curve
+        matrix[matrix<-1]=-1
+        matrix[matrix>1]=1
```

Comment:
Isn't this just undoing your 1.425 change?

Reply:
This does the clipping for values >1 or <-1, which is the max distance.